crates/sui-framework/packages/bridge/sources/limiter.move

```diff
@@ -65,6 +64,54 @@ module bridge::limiter {
         }
     }
 
+    struct UpdateRouteLimitEvent has copy, drop {
+        sending_chain: u8,
+        receiving_chain: u8,
+        new_limit: u64,
+    }
+
+    struct UpdateAssetPriceEvent has copy, drop {
+        token_id: u8,
+        new_price: u64,
+    }
+
+    // Abort if the route limit is not found
+    public fun get_route_limit(self: &TransferLimiter, route: &BridgeRoute): &u64 {
+        vec_map::get(&self.transfer_limits, route)
+    }
+
+    // Abort if the token's notional price is not found
+    public fun get_asset_notional_price(self: &TransferLimiter, token_id: &u8): &u64 {
+        vec_map::get(&self.notional_values, token_id)
+    }
+
+    public(friend) fun update_route_limit(self: &mut TransferLimiter, route: &BridgeRoute, new_usd_limit: u64) {
+        if (!vec_map::contains(&self.transfer_limits, route)) {
+            vec_map::insert(&mut self.transfer_limits, *route, new_usd_limit);
+            return
+        };
+        let entry = vec_map::get_mut(&mut self.transfer_limits, route);
+        *entry = new_usd_limit;
+        emit(UpdateRouteLimitEvent {
```

Comment:
This doesn't seem right to me -- the first time a route limit is updated no event will be emitted. I'm guessing the desire is to have it always emit this event...